task/buildah/0.1/buildah.yaml

```diff
@@ -159,23 +165,30 @@ spec:
       [ -n "$COMMIT_SHA" ] && LABELS+=("--label" "vcs-ref=$COMMIT_SHA")
       [ -n "$IMAGE_EXPIRES_AFTER" ] && LABELS+=("--label" "quay.expires-after=$IMAGE_EXPIRES_AFTER")
 
+      echo "$( date --utc -Ins ) Start buildah build"
       unshare -Uf $UNSHARE_ARGS --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -- buildah build \
         $VOLUME_MOUNTS \
         $BUILDAH_ARGS \
         ${LABELS[@]} \
         --tls-verify=$TLSVERIFY --no-cache \
         --ulimit nofile=4096:4096 \
+        --log-level=debug \
         -f "$dockerfile_path" -t $IMAGE $CONTEXT
 
+      echo "$( date --utc -Ins ) Start buildah from"
       container=$(buildah from --pull-never $IMAGE)
```

Comment:
need to add a `--log-level=debug` for the `buildah from` invocation as well .... in the slowest build from the last run in stage @jhutar I saw the time needed for this increase from 1 second with the fastest build to 10 seconds with the slowest build

Reply:
I suspect it is disk IO related like the other delays I've seen, but it would be good to confirm

Reply:
Oh, right! Adding and rerunning